app/frontend/Mail/Message/Recipient.svelte

```diff
@@ -74,9 +74,11 @@
     isPopupOpen = true;
   }
 
-  function addToAddressbook(addressbook: Addressbook) {
+  async function addToAddressbook(addressbook: Addressbook) {
     let person = recipient.createPerson();
     addressbook.persons.add(person);
+    person.addressbook = addressbook;
+    await person.save();
```

Comment:
This will only work for local address books. Maybe something like this? (untested)
```suggestion
    await person.moveToAddressbook(addressbook);
```